api/specs/web-server/_projects_nodes.py

```diff
@@ -44,36 +46,34 @@
     response_model=Envelope[NodeCreated],
     status_code=status.HTTP_201_CREATED,
 )
-def create_node(project_id: str, body: NodeCreate):
+def create_node(project_id: str, body: NodeCreate):  # noqa: ARG001
     ...
 
 
 @router.get(
     "/projects/{project_id}/nodes/{node_id}",
-    response_model=Envelope[NodeGet | NodeGetIdle],
-    # responses={"idle": {"model": NodeGetIdle}}, TODO: check this variant
+    response_model=Envelope[NodeGetIdle | NodeGetUnknown | DynamicServiceGet | NodeGet],
```

Comment:
IMO rest APIs should not have so many variants in inputs and/or outputs. Specially in the `projects` collection where there are so many inconsistencies in the interface. 

Can you discuss with me offline what is the rationale for this? thx

Reply:
as discussed created and linked an issue